tests/cli.rs

```diff
@@ -1,191 +1,197 @@
-extern crate assert_cli;
-#[macro_use]
-extern crate lazy_static;
-extern crate tempdir;
-
-use std::env;
-use std::str;
-use std::path::{Path, PathBuf};
-
-use tempdir::TempDir;
-
-lazy_static! {
-    static ref _CWD: PathBuf = env::current_dir().unwrap();
-    static ref CWD: &'static Path = _CWD.as_path();
-    // TODO test on release
-    static ref _BIN: PathBuf = CWD.join("target/debug/cobalt");
-    static ref BIN: &'static str = _BIN.to_str().unwrap();
-}
+extern crate assert_cmd;
+extern crate assert_fs;
+extern crate predicates;
+
+use std::process;
+
+use assert_cmd::prelude::*;
+use assert_fs::prelude::*;
+use predicates::prelude::*;
 
 #[test]
 pub fn invalid_calls() {
-    println!("Binary: {:?}", BIN.to_owned());
-    assert_cli::Assert::command(&[&BIN])
-        .fails()
-        .stderr()
-        .contains("requires a subcommand")
-        .unwrap();
-
-    assert_cli::Assert::command(&[&BIN, "--nonexistent-argument"])
-        .fails_with(1)
-        .stderr()
-        .contains(r"Found argument '--nonexistent-argument' which wasn't expected")
-        .unwrap();
+    process::Command::cargo_bin("cobalt")
+        .unwrap()
+        .assert()
+        .failure()
+        .stderr(&predicate::str::contains("requires a subcommand").from_utf8());
```

Comment:
I should add an `IntoOutput` that automatically converts `str` predicates.

See https://github.com/assert-rs/assert_cmd/issues/2